src/v1/chat_completion.rs

```diff
@@ -6,6 +6,7 @@ use crate::impl_builder_methods;
 use crate::v1::common;
 
 pub const GPT3_5_TURBO: &str = "gpt-3.5-turbo";
+pub const GPT3_5_TURBO_16K: &str = ""gpt-3.5-turbo-16k"";
```

Comment:
`""gpt-3.5-turbo-16k""` -> `"gpt-3.5-turbo-16k"` ?

Reply:
Oh sorry for that :)

Reply:
Fixed, thanks for your quick response.